### PR TITLE
Web interface for journal new features

### PR DESCRIPTION
**Pull request by Psycojoker**
See https://github.com/YunoHost/yunohost/pull/165 for more details

I haven't been able to find how I'm supposed to format javascript dates with mustache (strangly those strings are automatically transformed into `new Date()`).

I also haven't been able to handle to complex data structure of `metadata` that can include various things.

Screenshots:

2016-06-28-061750_1178x587_scrot

capture d ecran 2016-06-28 a 06 18 46

### CODE REVIEW

File: src/js/yunohost/controllers/tools.js [hunk 1]
```diff
@@ -95,6 +95,28 @@
         }
     });
 
+    // Display journals list
+    app.get('#/tools/journals', function (c) {
+        c.api("/journals", function(categories) {
+            c.view('tools/tools_journals', {
+                "data": categories,
+                formatDate: function() {
+                    console.log("caca");
```
Comment (zamentur) [inline src/js/yunohost/controllers/tools.js]: Don't forget to remove that.
I don't know how manage date with this framework

**Comment by zamentur** [comment]
log_list

log_display

**Comment by zamentur** [comment]
toomany

extrait

**Comment by zamentur** [comment]
I think it's mergeable, I have the idea for a lot of improvement, but this one is already ok.

**Comment by Psycojoker** [comment]
There are quite some conflict that prevent merging :/

I think it's always better to merge progressively and add new features along the road so that's a good news :)

Thanks for your work :heart:

**Comment by zamentur** [comment]
I have merge and retested it quickly seems good

**Comment by alexAubin** [comment]
In eee196e090250b04212f299cd13907097a8538ab, I propose to use relative time when listing logs (c.f. screenshot below) instead of the raw date which was used.

Doing this, I noticed that for relative time and formatted time (used on the page to visualize a specific log) the date/time was always in english. This is due to the internationalization mechanism of handlerbars. So far the solution I found was to wrap the view file with a `{{#intl}}` block to which I feed the locale currently used... There might be a more efficient solution for this :s 

2018-07-23-191202_1366x768_scrot

**Comment by alexAubin** [comment]
In 04582fd, I propagated what I talked about in https://github.com/YunoHost/yunohost/pull/165#issuecomment-406813565 : the share on yunopaste is done in the same way as via the command line so that both interface yield the same format for the log

**Pull request merged**